src/main/resources/errors.yaml

```diff
@@ -249,6 +240,17 @@ request-errors:
       
       ${SNIPPET.RESEND_USING_ONLY_DEFINED_COLUMNS}
 
+  - scope: FILTER
+    code: INVALID_FILTER_COLUMN_VALUES
+    title: Column value in filter is not valid
+    body: |-
+      Only values that are supported by the column data type can be included when filtering.
```

Comment:
Not 100% sure but this might be misleading. Not sure which of the problems occurs here:

1. Filter refers to columns of types we do not support (like Counter). If so, message is bit misleading
2. Filter refers to supported column but JSON type not supported. If so, message makes sense.

So just wanted to check which case we have here.

Reply:
Ok looking at test case(s), I think this is (2) so we are good. Sorry for the noise.